src/app.ts

```diff
@@ -25,25 +23,16 @@ const swaggerDefinition = {
     version: "1.0.0",
     description: "Documentation for the API endpoints",
   },
-  servers: [{ url: "http://localhost:3000/api/v1/auth" }],
+  servers: [{ url: "http://localhost:3000/api/v1" }],
```

Comment:
@MerlyneIradukunda . great job but To improve the codebase, it would be better to store this URL as an environment variable in the .env file. This will allow for easier configuration and flexibility in changing the URL in the future without modifying the code directly.